### ISSUE TITLE
semver understanding and breaking button changes

### ISSUE THREAD

**Issue by DimitarChristoff** [jgthms/bulma#763]
In view of recent 0.4.1 patch release, I'd like to question the logic behind some of that release.

`button.is-disabled` is now gone and shifted to `[disabled]` only - 
https://github.com/jgthms/bulma/blob/0.4.1/sass/elements/button.sass#L166
https://github.com/jgthms/bulma/blob/0.4.0/sass/elements/button.sass#L143-L144

this is a __breaking change__. it's not a __patch__, though semantically it makes more sense not to encourage people to rely on `pointer-events` to disable a button so this is a bug fix. even so, it's a breaking change.

given npm and people's liberal use of ^ when installing, projects depending on ^0.4.0 or ~0.4.0 suddenly got the breaking changes and started to build wrong / with side effects in CI. 

and finally. https://github.com/jgthms/bulma/blob/master/CHANGELOG.md - no mention of button itself.

**Comment by worldspawn** [comment]
Yeah this has just screwed me over. I'm also seeing different alignments in .radio/.checkbox but thats less breaky than this. This should not happen stepping up a point release.

**Comment by DimitarChristoff** [comment]
in 0.4.3 for example, `.content figure` suddently got a `text-align: center` . breaking again.

**Comment by jgthms** [comment]
Bulma hasn’t been following semver, and is not advertised as so yet. It’s sometimes hard to figure what constitutes a breaking change and what doesn’t. Some improvements can be overrides by customisation. Some fixes are reverting parts that shouldn’t have been there in the first place.

Up until `1.0.0` there can be “breaking changes”. After that we need to define what exactly is one.
